python/instrumentation/openinference-instrumentation-llama-index/examples/chat_engine_stream_chat.py

```diff
@@ -26,6 +26,6 @@
 Settings.llm = OpenAI(model="gpt-3.5-turbo")
 
 if __name__ == "__main__":
-    query_engine = index.as_query_engine(streaming=True, similarity_top_k=1)
-    response_stream = query_engine.query("What did the author do growing up?")
+    query_engine = index.as_chat_engine()
+    response_stream = query_engine.stream_chat("What did the author do growing up?")
```

Comment:
```suggestion
    chat_engine = index.as_chat_engine()
    response_stream = chat_engine.stream_chat("What did the author do growing up?")
```

Is this still a query engine? Maybe we should rename the variable?

Reply:
you're right, my oversight. thx!